From 151f23476a1323f30da55bcf18c29938dc0ed425 Mon Sep 17 00:00:00 2001
From: Chad Dupuis <cdupuis@redhat.com>
Date: Thu, 21 Aug 2014 13:06:53 -0400
Subject: [ethernet] qlcnic: remove duplicate QLC_83XX_GET_LSO_CAPABILITY define

Message-id: <1408626422-3941-130-git-send-email-cdupuis@redhat.com>
Patchwork-id: 88651
O-Subject: [PATCH RHEL 7.1 129/138] qlcnic: remove duplicate QLC_83XX_GET_LSO_CAPABILITY define
Bugzilla: 1089352
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Dan Carpenter <dan.carpenter@oracle.com>

Bug 1089352 (https://bugzilla.redhat.com/show_bug.cgi?id=1089352)

net commit id 5e3ec11b642e2e50484dce5ef2500f5aca47a40f

The QLC_83XX_GET_LSO_CAPABILITY define is cut and pasted twice so we can
delete the second instance.

Signed-off-by: Dan Carpenter <dan.carpenter@oracle.com>
Acked-by: Sony Chacko <sony.chacko@qlogic.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/qlogic/qlcnic/qlcnic_83xx_hw.h b/drivers/net/ethernet/qlogic/qlcnic/qlcnic_83xx_hw.h
index 97784d0..2bf101a 100644
--- a/drivers/net/ethernet/qlogic/qlcnic/qlcnic_83xx_hw.h
+++ b/drivers/net/ethernet/qlogic/qlcnic/qlcnic_83xx_hw.h
@@ -418,7 +418,6 @@ enum qlcnic_83xx_states {
 #define QLC_83XX_GET_FUNC_MODE_FROM_NPAR_INFO(val) (val & 0x80000000)
 #define QLC_83XX_GET_LRO_CAPABILITY(val)  (val & 0x20)
 #define QLC_83XX_GET_LSO_CAPABILITY(val)  (val & 0x40)
-#define QLC_83XX_GET_LSO_CAPABILITY(val)  (val & 0x40)
 #define QLC_83XX_GET_HW_LRO_CAPABILITY(val)  (val & 0x400)
 #define QLC_83XX_GET_VLAN_ALIGN_CAPABILITY(val) (val & 0x4000)
 #define QLC_83XX_GET_FW_LRO_MSS_CAPABILITY(val) (val & 0x20000)
-- 
1.7.1